src/Standards/Generic/Sniffs/ControlStructures/DisallowYodaConditionsSniff.php

```diff
@@ -110,14 +105,14 @@ public function process(File $phpcsFile, $stackPtr)
                     return;
                 }
 
-                // If there is nothing inside the parenthesis, it it not a Yoda.
+                // If there is nothing inside the parenthesis, it is not a Yoda.
```

Comment:
```suggestion
                // If there is nothing inside the parenthesis, it is not a Yoda condition.
```

(if we're updating the comment now anyway, let's make it grammatically correct)